### PR TITLE
Add more OS platforms for wazuh-chef Kitchen tests

### LINKED ISSUE #439: Use new Docker images in kitchen (Chef) tests to get Systemd activated
Hi all!

We need to change the Docker images we use in `Kitchen` tests so we get `Systemd` which fits better with Kitchen tests when checking the status of the service, or if it's enabled or disabled, etc .. . 

**Tasks**

- [x] Identify the proper Docker images.
- [x] Adapt `kitchen.yml` to use ENVIRONMENT variables.
- [x] Adapt `run.sh`
- [x] Adapt the corresponding tests.
- [x] Verify the changes
- - [x] Ubuntu
- - - [x]  14.04
- - - [x] 16.04
- - - [x] 18.04

- [x] CentOS 7.

- [x] Amazon Linux 2.

Kr,

Rshad

### PR DESCRIPTION
**Pull request by rshad**
Hi all!

This PR resolves #439.

Kr,

Rshad

### CODE REVIEW

File: kitchen/wazuh-chef/Dockerfile [hunk 1]
```diff
@@ -27,6 +18,14 @@ RUN cd $HOME && \
     cd $HOME/wazuh-qa/kitchen/wazuh-chef/manager/files/ && \
     bundle install
 
+RUN git clone https://github.com/rshad/kitchen-docker && \
```
Comment (manuasir) [inline kitchen/wazuh-chef/Dockerfile]: This shouldn't be in any personal repo, please move it to a Wazuh repository.

File: kitchen/wazuh-chef/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
-COPY entrypoint.sh /usr/local/bin/
-
-RUN chmod 777 /usr/local/bin/entrypoint.sh \
-    && ln -s /usr/local/bin/entrypoint.sh /
-
-ENTRYPOINT ["entrypoint.sh"]
-
 RUN apt-get update && apt install docker.io git curl wget -y
 
 RUN apt-get update
```
Comment (xr09) [inline kitchen/wazuh-chef/Dockerfile]: This line is not needed here.

File: kitchen/wazuh-chef/Dockerfile [hunk 3]
```diff
@@ -1,14 +1,5 @@
...
-CMD tail -f /dev/null
-
-COPY entrypoint.sh /usr/local/bin/
-
-RUN chmod 777 /usr/local/bin/entrypoint.sh \
-    && ln -s /usr/local/bin/entrypoint.sh /
-
-ENTRYPOINT ["entrypoint.sh"]
-
 RUN apt-get update && apt install docker.io git curl wget -y
```
Comment (xr09) [inline kitchen/wazuh-chef/Dockerfile]: Clean APT database after installing packages to reduce layer size.

File: kitchen/wazuh-chef/Dockerfile (hunk 1 above)
Comment (rshad) [reply kitchen/wazuh-chef/Dockerfile]: Hi @manuasir !

Now we have the fixes for `kitchen-docker` in this same branch.

Kr,

Rshad

**Pull request merged**